src/Command/ModelCommand.php

```diff
@@ -1444,4 +1454,94 @@ protected function possibleEnumFields(TableSchemaInterface $schema): array
 
         return $fields;
     }
+
+    /**
+     * @param \Cake\Database\Schema\TableSchemaInterface $schema
+     * @return array<string, mixed>
+     */
+    protected function getEnumDefinitions(TableSchemaInterface $schema): array
```

Comment:
Should we make the schema parsing logic in `EnumCommand` public methods so that we can re-use it here? Having two implementations of it could diverge.

Reply:
I moved it into own utility method (static) for reuse